gm4_tower_structures/data/gm4_guidebook/advancements/tower_structures/page_0.json

```diff
@@ -17,239 +17,230 @@
   },
   "parent": "gm4_guidebook:root",
   "criteria": {
-    "requirement_1": {
-      "trigger": "minecraft:player_generates_container_loot",
+    "badlands": {
+      "trigger": "minecraft:location",
       "conditions": {
```

Comment:
This file is missing the necessary guidebook `load.status` score checks, which causes the unlock page message to be displayed in chat even when the guidebook module is not enabled.